packages/cf-component-input/src/Input.js

```diff
@@ -19,14 +19,7 @@ class Input extends React.Component {
     }
 
     return (
-      <input
-        type={this.props.type}
-        className={className}
-        name={this.props.name}
-        value={this.props.value}
-        onChange={this.handleChange}
-        placeholder={this.props.placeholder}
-        autoComplete={this.props.autoComplete}/>
+      <input className={className} onChange={this.handleChange} {...this.props} />
```

Comment:
I ran the demo page as a testing playground and this breaks the input. 

This seemed to fix
```jsx
  render() {
    let { className, ...props } = this.props;

    className = className + ' cf-input cf-input--' + props.type;

    if (props.invalid) {
      className += ' cf-input--invalid';
    }

    return (
      <input {...this.props} className={className} onChange={this.handleChange}  />
    );
  }
```